ingest-app/src/cmr/ingest/services/subscriptions_helper.clj

```diff
@@ -29,6 +31,14 @@
 ;; SET NEXT_FIRE_TIME =(((cast (SYS_EXTRACT_UTC(SYSTIMESTAMP) as DATE) - DATE'1970-01-01')*86400 + 1200) * 1000)
 ;; WHERE trigger_name='EmailSubscriptionProcessing.job.trigger';
 
+(defn normalize-subscription-search
```

Comment:
There is no need for this function. We should work the `string/trim` into the existing function `normalize-parameters-v1`, and just call that function when needed.

Reply:
The idea with this function is that it serves to be a constant call for the future, even if we want to change the normalization algorithm with a future `normalize-parameters-v2`. That way, we could just plug the new v2 into this function without having to change all the function calls in code/tests.

Reply:
I don't follow your argument. :) If we change the normalization algorithm in the future, we would just change the function `normalize-parameters-v1`. The test should go with that function rather than this one. I still recommend delete this function and move the tests to `normalize-parameters-v1`. Since we can't have more than one algorithms to normalize parameter, change the name of the function to `normalize-parameters`.

Reply:
After discussion with Thomas and Leo, decided to collapse to just one function, `normalize parameters`.